docs/src/man/working_with_dataframes.md

```diff
@@ -407,6 +407,11 @@ julia> df[in.(df.A, Ref([1, 5, 601])), :]
    3 │   601      7    301
 ```
 
+The `Ref` wrapper to `[1, 5, 601]` is needed to protect the vector against being
+broadcasted over (the vector will be treated as a scalar when wrapped in `Ref`).
+Alternatively you could use a comprehension like this:
+`[a in [1, 5, 601] for a in df.A]`.
```

Comment:
But this is slower as it recreates the array, right?

Reply:
Yes:
```
julia> length(x)
1000000

julia> @time [a in [1, 5, 601] for a in x];
  0.066654 seconds (1.06 M allocations: 80.224 MiB, 10.75% gc time, 32.44% compilation time)

julia> in.(x, Ref([1, 5, 601]));

julia> @time in.(x, Ref([1, 5, 601]));
  0.002442 seconds (8 allocations: 126.547 KiB)

julia> @time in([1, 5, 601]).(x);
  0.002134 seconds (8 allocations: 126.531 KiB)
```
I will add this comment.

Reply:
added